README.md

```diff
@@ -24,42 +31,46 @@ This version of the library is compatible with .NET Framework version 4.5 and ab
 
 2. Import the package (note that the namespace is different from the package name):
 
-        using LaunchDarkly.Client.Redis;
+        using LaunchDarkly.Sdk.Server.Integrations;
 
-3. When configuring your `LDClient`, add the Redis feature store:
+3. When configuring your `LdClient`, add the Redis data store as a `PersistentDataStore`. You may specify any custom Redis options using the methods of `RedisDataStoreBuilder`. For instance, to customize the Redis URI:
 
-        Configuration ldConfig = Configuration.Default("YOUR_SDK_KEY")
-            .WithFeatureStoreFactory(RedisComponents.RedisFeatureStore());
-        LdClient ldClient = new LdClient(ldConfig);
+        var ldConfig = Configuration.Default("YOUR_SDK_KEY")
+            .DataStore(
+                Components.PersistentDataStore(
+                    Redis.DataStore().Uri("redis://my-redis-host")
+                )
+            )
+            .Build();
+        var ldClient = new LdClient(ldConfig);
 
-4. Optionally, you can change the Redis configuration by calling methods on the builder returned by `RedisFeatureStore()`:
+By default, the store will try to connect to a local Redis instance on port 6379.
 
-        Configuration ldConfig = Configuration.Default("YOUR_SDK_KEY")
-            .WithFeatureStoreFactory(
-                RedisComponents.RedisFeatureStore()
-                    .WithRedisHostAndPort("my-redis-host", 6379)
-                    .WithConnectTimeout(TimeSpan.FromSeconds(3))
-            );
-        LdClient ldClient = new LdClient(ldConfig);
+## How the SDK uses Redis
```

Comment:
@bwoskow-ld - This new section is something I'm proposing to add to the docs for each database integration. Without getting into details of exactly how a flag is serialized— since that's the SDK's business, and isn't specific to any one database integration— I think it's a good idea for us to document things like "it's going to create such-and-such keys in your database".

However, it might be that this content would be a better fit for the `docs.launchdarkly.com` persistent data store page. In fact, given the amount of information per database, maybe this would be a good time to create subpages there for Redis, Consul, and DynamoDB.

Reply:
I do think that this information is better placed on `docs.launchdarkly.com` since it'll be the same content for the Redis integrations (and likewise, Consul and DymanoDB) regardless of the SDK. Also, I'm a bit concerned about the split of information between here and the docs site; ideally we wouldn't require users to have to look at both to get a complete picture of how the Redis integration works.

cc @scribblingfox -- do you have a preference if we add this kind of content now or wait for the SDK docs refresh? What's the latest ETA for that project to start?

Reply:
@bwoskow-ld Let's put the content in now, and I agree with the preference to put it on the  docs site, rather than the readme. 

SDK docs refresh is the next strategic project after the IA thing I'm working on right now.

Reply:
I've removed it from this PR because it's already being reviewed in https://github.com/launchdarkly/git-gatsby/pull/457.